src/common/low_precision_transformations/tests/fold_fake_quantize_in_transformations.cpp

```diff
@@ -67,21 +67,34 @@ class FoldFakeQuantizeInTransformations
 
         const auto params = TestTransformationParams(testValues.params).setUpdatePrecisions(testValues.updatePrecision);
 
-        const auto constant = std::make_shared<ngraph::opset1::Constant>(testValues.actual.constPrecision,
-                                                                         testValues.constShape,
-                                                                         testValues.actual.constValues);
+        std::shared_ptr<ngraph::Node> dataSource;
+        std::shared_ptr<ngraph::opset1::Parameter> parameter;
+        bool useParameterAsDataSource = (testValues.actual.constValues.size() == 0);
+
+        if (useParameterAsDataSource) {
+            // for data-independent const folding
+            parameter =
+                std::make_shared<ngraph::opset1::Parameter>(testValues.actual.constPrecision, testValues.constShape);
+            dataSource = parameter;
+        } else {
+            dataSource = std::make_shared<ngraph::opset1::Constant>(testValues.actual.constPrecision,
+                                                                    testValues.constShape,
+                                                                    testValues.actual.constValues);
+        }
 
```

Comment:
to make sure the test hits the dot-interval const-fold case rather than generic FQ-const-fold, we must use parameter as data input rather than const